integration/pmem/pmem_test.sh

```diff
@@ -50,6 +52,25 @@ function setup() {
 	fi
 }
 
+function ctr_exec {
+	args=()
+	for i; do
+		if [[ $i = *" "* ]]; then
+			args+=(\""${i}"\")
+		else
+			args+=("${i}")
+		fi
```

Comment:
You could just always quote to simplify this (that would also protect against tab and not just space):

```bash
# Always quote to retain whitespace
args+=(\""${i}"\")
```